change/@azure-msal-angular-1424939a-f410-4eaf-b293-c874faf0adff.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update build:all:prod command",
```

You'll want to set this to patch so that our pipelines recognize a new version needs to be published
```suggestion
  "type": "patch",
  "comment": "Fix build output during publishing",
```